addon/initializers/ember-feature-flags.js

```diff
@@ -1,9 +1,12 @@
 import Ember from 'ember';
 import ifFeature from 'ember-feature-flags/helpers/if-feature';
+import features from '../features';
 
 export default {
   name: 'ember-feature-flags',
   initialize: function( container, application ) {
+    features.setup(Ember.merge({}, application.FEATURES));
```

Comment:
I'd prefer to not have this be default behavior. I think it would cause confusion if you were setting your features from a source other than the config because if this initializer ran after you setup your features, this would clobber them.